m4_sessions/1804inning/self_doubt.md

```diff
@@ -5,12 +5,40 @@ Luke Chambers
 Michael Gatewood
 Ricardo Viera
 
-Name of Session: 
+Name of Session: You Probably Won't Come To This Session, You'd Rather Be Working On Your Project
 
-Session Description (Why should students come to this session):
+Session Description (Why should students come to this session): Your project deadline is coming up, and you doubt you'll even finish it. It seems like everyone is ahead of you. You're probably going to repeat anyway. Why did you even come to Turing in the first place? You probably stopped reading this already, because we doubt this description is helpful at all, but come learn how self-doubt is impeding your learning 
 
-Session Outcomes (What will students learn/leave with?):
+Session Outcomes (What will students learn/leave with?): Students will leave with confidence to do something that they don't know how to do. Leave with strategies to cope with when doubt creeps.
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
 
+Step -1: Talk Guidelines --- Ricardo
+ - This may be a little sensitive for people. Please respect each other. Everyone has different opinions that come from different life experiences. 
+ 
+ https://www.wikihow.com/Become-a-Decent-Person
+
+Step 0: Intros --- Anna
+ - Count off to break people into groups (1..4) 
+ - Introduce yourself to your group (4 groups, we will each lead a group)
+  - Introduction question: 
+
+Step 1: Recognition --- Luke
+ - Why is it a problem (College Study on testing or some other stuff that we don't know the name of yet)
+ - What are some things you tell yourself when you begin to self doubt? What situations do you find yourself self doubting in? Did those thoughts help the situation or hurt the situation or were they neutral?
+ - Write down expressions of self doubt on the board
+
+Step 3, 4, 5, and 6: Group Leads
+ - Doubt you will ever be a programmer --- Michael
+ - Doubt you will be as good as the people around you --- Luke
+ - Doubt you will pass the module --- Ricardo
+ - Doubt you are ready to start a job --- Anna
+ 
+Step 3.5 and 4.5 and 5.5 and 6.5: Discuss strategies to address the self doubt in the proposed situation
+ - At the end, write your solutions on the white board
+ 
+ Step Idk maybe 9?: Wrap-Up --- Michael
+  - Address solutions written on the board
+  - Open up discussion to the room
+
```

Comment:
This looks great!!  I might just add timing (how long is each step) and absolutely do a wrap up.  Usually that is the most important thing for people to actually synthesize.